posthog/celery.py

```diff
@@ -413,10 +413,12 @@ def graphile_worker_queue_size():
 
         # Track the number of jobs that will still be run at least once or are currently running based on job type (i.e. task_identifier)
         # Completed jobs are deleted and "permanently failed" jobs have attempts == max_attempts
+        # Jobs not yet eligible for execution are filtered out with run_at <= now()
         cursor.execute(
             """
-        SELECT task_identifier, count(*) as c, EXTRACT(EPOCH FROM MIN(updated_at)) as oldest FROM graphile_worker.jobs
+        SELECT task_identifier, count(*) as c, EXTRACT(EPOCH FROM MIN(run_at)) as oldest FROM graphile_worker.jobs
         WHERE attempts < max_attempts
+        AND run_at <= now()
```

Comment:
nit: One thing that could bite us here is timezones, do we ensure the db timezone is used when writing to this table?

Reply:
Good catch. This column in a `timestampz`, so I'm assuming this should work regardless of the connection's TZ?
![image](https://user-images.githubusercontent.com/6241083/224681722-6e237e9f-a09f-419a-bf36-d3291578e7a5.png)